src/main/java/org/apache/sling/feature/extension/content/ContentOrderMergeProcessor.java

```diff
@@ -51,6 +51,9 @@ public boolean canMerge(Extension extension) {
 
     @Override
     public void merge(HandlerContext context, Feature target, Feature source, Extension targetEx, Extension sourceEx) {
+        if (target == null) {
+            return;
```

Comment:
Sorry - I was wrong. Instead of just returning, it should be:
```
if (target == null) {
    target.getExtensions().add(sourceEx)
    return;
}
```